pages/fantoir/index.js

```diff
@@ -0,0 +1,124 @@
+import {useState} from 'react'
+import PropTypes from 'prop-types'
+import Link from 'next/link'
+import {sortBy, debounce} from 'lodash'
+
+import {getCounties} from '@/lib/api-geo'
+
+import {Database} from 'react-feather'
+
+import Page from '@/layouts/main'
+import Head from '@/components/head'
+import Section from '@/components/section'
+import SearchBar from '@/components/search-bar'
+
+import theme from '@/styles/theme'
+const title = 'Fantoir'
+
+function Fantoir({counties}) {
+  const sortedCounties = sortBy(counties, ['nom'])
```

Comment:
Les départements ne changeront pas, on pourrait optimiser en utilisant un `useMemo` ici.
```suggestion
const sortedCounties = useMemo(() => {
    return sortBy(counties, ['nom'])
  }, [counties])```